src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -631,15 +631,26 @@ private function computeDoc(bool $v3, Documentation $documentation, \ArrayObject
         $security = [];
 
         if ($this->oauthEnabled) {
-            $securityDefinitions['oauth'] = [
-                'type' => $this->oauthType,
-                'description' => 'OAuth client_credentials Grant',
-                'flow' => $this->oauthFlow,
+            $oauthAttributes = [
                 'tokenUrl' => $this->oauthTokenUrl,
                 'authorizationUrl' => $this->oauthAuthorizationUrl,
                 'scopes' => $this->oauthScopes,
             ];
 
+            $securityDefinitions['oauth'] = [
+                'type' => $this->oauthType,
+                'description' => 'OAuth client_credentials Grant',
```

Comment:
Not sure why this is hardcoded. Can you fix this as well?

Reply:
@teohhanhui IMO, as this description is not yet configurable, we should change it in an other PR on master, because change this should be a new feature ? WDYT ?

Reply:
It doesn't need to be configurable, but we should generate the correct description based on the flow, no?

Reply:
Oh sure :sweat_smile: 

Something like this ?

```suggestion
                'description' => 'This API uses OAuth 2 with the ' . $this->oauthFlow . ' grant flow.',
```